src/components/PopUpTrackInfo.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import PopupDialog from './PopupDialog.js';
+
+export const PopUpTrackInfo = ({
+  open,
+  attributes,
+  close,
+}) => {
+    return(
+      <div>
+        <PopupDialog open={open} close={close}>
+            <h5>Object Information</h5>
+            <table>
+              {/* Node info here */}
+              {(attributes || []).map(function(attribute){
+                return <tr key={attribute[0]} style={{border: '1px solid black'}}>
```

Comment:
It looks like React doesn't like simple tables without a `<tbody>` tag. I'm getting it throwing up a warning in the console when the table renders:

```
Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.
tr
table
div
div
CardBody@http://127.0.0.1:3001/static/js/bundle.js:148921:19
div
Card@http://127.0.0.1:3001/static/js/bundle.js:148822:19
div
Container@http://127.0.0.1:3001/static/js/bundle.js:151747:19
div
div
./node_modules/reactjs-popup/dist/reactjs-popup.esm.js/Popup<@http://127.0.0.1:3001/static/js/bundle.js:146711:22
div
PopupDialog@http://127.0.0.1:3001/static/js/bundle.js:3707:14
div
```

It looks like the browser automatically creates a `<tbody>` if the table doesn't have one, and React demands that you not let it do that and actually create the `<tbody>` manually instead, so that the browser isn't adding markup that your code doesn't know about.

We had better do what Facebook says I guess, I don't think there's a way to turn that warning off.